README.md

```diff
@@ -235,6 +238,28 @@ Here's what the full _default_ module configuration looks like:
 }
 ```
 
+```
+### Using a different storage driver
```

Comment:
I suggest that this should be a sub-chapter of configuration, so use: #### instead

Reply:
updated!